SwiftGit2/Repository.swift

```diff
@@ -237,61 +237,61 @@ final public class Repository {
 	/// oid - The OID of the blob to look up.
 	///
 	/// Returns the blob if it exists, or an error.
-	public func blobWithOID(_ oid: OID) -> Result<Blob, NSError> {
-		return self.withLibgit2Object(oid, type: GIT_OBJ_BLOB) { Blob($0) }
+	public func blob(withIdentity oid: OID) -> Result<Blob, NSError> {
+		return mapGitObject(withIdentity: oid, type: GIT_OBJ_BLOB) { Blob($0) }
 	}
 	
 	/// Loads the commit with the given OID.
 	///
 	/// oid - The OID of the commit to look up.
 	///
 	/// Returns the commit if it exists, or an error.
-	public func commitWithOID(_ oid: OID) -> Result<Commit, NSError> {
-		return self.withLibgit2Object(oid, type: GIT_OBJ_COMMIT) { Commit($0) }
+	public func commit(withIdentity oid: OID) -> Result<Commit, NSError> {
+		return mapGitObject(withIdentity: oid, type: GIT_OBJ_COMMIT) { Commit($0) }
 	}
 	
 	/// Loads the tag with the given OID.
 	///
 	/// oid - The OID of the tag to look up.
 	///
 	/// Returns the tag if it exists, or an error.
-	public func tagWithOID(_ oid: OID) -> Result<Tag, NSError> {
-		return self.withLibgit2Object(oid, type: GIT_OBJ_TAG) { Tag($0) }
+	public func tag(withIdentity oid: OID) -> Result<Tag, NSError> {
+		return mapGitObject(withIdentity: oid, type: GIT_OBJ_TAG) { Tag($0) }
 	}
 	
 	/// Loads the tree with the given OID.
 	///
 	/// oid - The OID of the tree to look up.
 	///
 	/// Returns the tree if it exists, or an error.
-	public func treeWithOID(_ oid: OID) -> Result<Tree, NSError> {
-		return self.withLibgit2Object(oid, type: GIT_OBJ_TREE) { Tree($0) }
+	public func tree(withIdentity oid: OID) -> Result<Tree, NSError> {
```

Comment:
Hmm… This is tricky. At a minimum, I think this should be only `tree(with oid: OID)`.

Here are some other options:

```swift
load(tree oid: OID)
fetch(tree oid: OID)
tree(_ oid: OID)
```

Do any of those seem better to you?
```